---
title: "Rapid method for generating designer algal mitochondrial genomes"
collection: publications
permalink: /publication/2019-12-19-Rapid-method-for-generating-designer-algal-mitochondrial-genomes
excerpt: 'This paper demonstrates the methods for cloning and engineering of mitochondrial genomes from Phaeodactylum tricornutum. The mitochondrial genome was cloned in yeast and bacteria and found no significant host burden. Following research will detail methods for mitochondrial DNA delivery to P. tricornutum. Together, these techonlogies will estalish P. tricornutum as a model system for studying the mitochondria of diatoms and allow easy installation of biosynthetic pathways.'
date: 2019-12-19
venue: 'bioRxiv'
paperurl: 'http://rcochrane.github.io/files/paper2.pdf'
citation: 'RR. Cochrane, SL. Brumwell, MPM. Soltysiak, S. Hamadache, JG. Davis, J. Wang, SQ. Tholl, P. Janakirama, DR. Edgell, BJ. Karas. (2019). &quot;Rapid method for generating designer algal mitochondrial genomes.&quot; <i>bioRxiv</i>.'
---
**ABSTRACT** With synthetic biology, we can turn algae into bio-factories that produce high-value molecules, such as medicines or biofuels, or tackle global challenges, such as malnutrition or climate change. This realization has provoked rapid progress towards the creation of genetic tools for multiple algal species, notably _Phaeodactylum tricornutum_. The power of synthetic biology to generate more useful or productive organisms is contingent on the ability to produce diverse DNA molecules and rapidly screen them for beneficial variants. However, it is still relatively expensive to synthesize DNA, and delivering large DNA to eukaryotic cellular compartments remains challenging. In this study, we establish a robust system for building algal mitochondrial genomes as a practical alternative to DNA synthesis. Our approach permits an inexpensive and rapid generation of mitochondrial derivatives designed for testing targeted DNA delivery. First, we cloned the mitochondrial genome of _P. tricornutum_ into the eukaryotic host organism _S. cerevisiae_ using two different techniques, transformation-associated recombination and PCR-based cloning. Next, we analyzed the cloned genomes by multiplex PCR, and correct genomes were transferred to prokaryotic host _E. coli_. Then these genomes were again analyzed by multiplex PCR, followed by diagnostic digest and complete-plasmid sequencing to evaluate the fidelity of each cloning method. Finally, we assessed the burden on eukaryotic and prokaryotic hosts to propagate the cloned genomes. We conclude that our system can reliably generate variants for genome-level engineering of algal mitochondria.

<br/><img src='https://rcochrane.github.io/images/paper1coverphoto.png'>

# Figures and Tables

<br/><img src='https://rcochrane.github.io/images/paper1figure1.png'>
**Figure 1.** Design-Build-Test cycle for the rapid engineering of organelle genomes. **A)** Genomes will be designed based on existing knowledge and discoveries from previous cycles; in the build stage, genomes will be synthesized, then assembled and cloned in yeast; in the test stage, genomes will be isolated from yeast, moved to an intermediate prokaryotic host and delivered directly or indirectly to the destination organelle compartment to test for viability, function, and localization. **B-C)** Schematic of the approaches used to clone the mitochondrial genomes of _P. tricornutum_. **B)** TAR cloning method; **C)** PCR-based cloning method. Step I differs between the PCR and TAR cloning methods where, for the PCR method, multiple overlapping fragments are amplified, while for the TAR cloning method, genomic DNA is linearized at a specific location. For both methods, the plasmid backbone contains homology overlaps to the appropriate fragments or location in the genome. Steps II-IV are the same for both methods including DNA transformation and assembly in _S. cerevisiae_ via homologous recombination, and transfer of clone genomes into _E. coli_.

**Table 1.** Cloning of the _P. tricornutum_ mitochondrial genome in the host organisms _S. cerevisiae_ and _E. coli_. Two PCR-cloning assemblies and one TAR-cloning assembly were performed. For the _E. coli selection_ media, CM indicates chloramphenicol antibiotic.
<br/><img src='https://rcochrane.github.io/images/paper1table1.png'>

<br/><img src='https://rcochrane.github.io/images/paper1figure2.png'>
**Figure 2.** Design, amplification, and analyses of cloned _P. tricornutum_ mitochondrial genomes. **A)** Plasmid maps of _P. tricornutumâ€™s_ mitochondrial genomes cloned using PCR or TAR cloning methods. For the PCR-based cloning method, the relative sizes and positions of the eight mitochondrial fragments and the two plasmid backbone fragments are shown in blue and orange, respectively. For the TAR-based cloning method, the relative sizes and positions of the linearized genome and the two plasmid backbone fragments are shown in blue and orange, respectively. In addition, the six MPX PCR amplicons used for diagnostic screening are indicated in green. These images were generated using Geneious version 2020.0, created by Biomatters. **B)** Agarose gel electrophoresis of the 10 PCR amplified fragments of the _P. tricornutum_ mitochondrial genome for PCR-based cloning including the pAGE3.0 backbone. The resulting amplicon sizes for Fragments 1 to 10 are 5217, 5190, 5197, 5175, 2765, 9124, 8571, 5605, 5299, and 9264 bp, respectively. **C)** MPX PCR screen of six cloned algal mitochondrial genomes isolated from _E. coli_. The expected size for each MPX amplicon is indicated by its name. **D)** Diagnostic restriction digest of six cloned algal mitochondrial genomes using SrfI and SacII. For the pPT-PCR clones, the expected band sizes are 39925, 8758, 7177, 3276, and 707 bp; and for the pPT-TAR clones, the expected band sizes are 70647, 11506, 8758, 3276, and 707 bp.

**Table 2** Summary of mutations identified in the cloned _P. tricornutum_ mitochondrial plasmids. Identified mutations are categorized as point mutations or, gap mutations.
<br/><img src='https://rcochrane.github.io/images/paper1table2.png'>

<br/><img src='https://rcochrane.github.io/images/paper1fig3.png'>
**Figure 3.** Analysis of growth of _S. cerevisiae_ and _E. coli_ strains harbouring cloned _P. tricornutum_ mitochondrial genomes on solid media and in liquid media. **A)** Dilutions of _S. cerevisiae_ strains plated on solid synthetic complete media lacking histidine. **B)** Dilutions of _E. coli_ strains plated on solid LB media supplemented with chloramphenicol only or with chloramphenicol and arabinose to increase plasmid copy number. **C)** Growth curves of _S. cerevisiae_ strains grown in liquid synthetic complete media lacking histidine. **D)** Growth curves of _E. coli_ grown in liquid LB media supplemented with chloramphenicol only or with chloramphenicol and arabinose. Td - doubling time.

Recommended citation: RR. Cochrane, SL. Brumwell, MPM. Soltysiak, S. Hamadache, JG. Davis, J. Wang, SQ. Tholl, P. Janakirama, DR. Edgell, BJ. Karas. (2019). "Rapid method for generating designer algal mitochondrial genomes." <i>bioRxiv</i>. 0.
