---
title: "Designer Sinorhizobium meliloti strains and multi-functional vectors enable direct inter-kingdom DNA transfer"
collection: publications
permalink: /publication/2019-06-17-Designer-Sinorhizobium-meliloti-strains-and-multi-functional-vectors-enable-direct-inter-kingdom-DNA-transfer
excerpt: 'This paper establishes S. meliloti as a effective microbial host to GC-rich DNA. This research will enable manipulation of soil microbiomes, and cloning of previously difficult DNA.'
date: 2019-06-17
venue: 'PLoS One'
paperurl: 'https://ryancochrane.github.io/files/paper1.pdf'
citation: 'SL. Brumwell, MR. MacLeod, T. Huang, RR. Cochrane, RS. Meaney, M. Zamani, O. Matysiakiewicz, KN. Dan, P. Janakirama, DR. Edgell, TC. Charles, TM. Finan, BJ. Karas. (2019). &quot;Designer Sinorhizobium meliloti strains and multi-functional vectors enable direct inter-kingdom DNA transfer.&quot; <i>PLoS ONE</i>. 14(7): e0219562.'
---
**ABSTRACT** Storage, manipulation and delivery of DNA fragments is crucial for synthetic biology applications, subsequently allowing organisms of interest to be engineered with genes or pathways to produce desirable phenotypes such as disease or drought resistance in plants, or for synthesis of a specific chemical product. However, DNA with high GC content can be unstable in many host organisms including _Saccharomyces cerevisiae_. Here, we report the development of _Sinorhizobium meliloti_, a nitrogen-fixing plant symbiotic alpha-Proteobacterium, as a novel host that can store DNA, and mobilize DNA to _E. coli_, _S. cerevisiae_, and the eukaryotic microalgae _Phaeodactylum tricornutum_. To achieve this, we deleted the hsdR restriction-system in multiple reduced genome strains of _S. meliloti_ that enable DNA transformation with up to 1.4 x 10<sup>5</sup> and 2.1 x 10<sup>3</sup> CFU μg<sup>-1</sup> of DNA efficiency using electroporation and a newly developed polyethylene glycol transformation method, respectively. Multi-host and multi-functional shuttle vectors were constructed and stably propagated in _S. meliloti_, _E. coli_, _S. cerevisiae_, and _P. tricornutum_. We also developed protocols and demonstrated direct transfer of these multi-host shuttle vectors via conjugation from _S. meliloti_ to _E. coli_, _S. cerevisiae_, and _P. tricornutum_. The development of _S. meliloti_ as a new host for inter-kingdom DNA transfer will be invaluable for synthetic biology research and applications, including the installation and study of genes and biosynthetic pathways into organisms of interest in industry and agriculture.

<br/><img src='https://ryancochrane.github.io/images/paper2coverphoto.png'>



# Figures and Tables

<br/><img src='https://ryancochrane.github.io/images/paper2figure1.png'>
**Figure 1. Development of designer _S. meliloti_ strains and multi-functional vectors. (a)** Schematic of the designer _S. meliloti_ ΔpSymA ΔhsdR strains including the genome reduction and deletion of the restriction-system. Several versions of the designer strains were developed including strains with the pTA-Mob conjugative plasmid, and/or compatible genome engineering vectors, pAGE or pBGE, installed. **(b)** Vector map of pAGE/pBGE MHS vector with both standard and interchangeable components. Standard components include the BAC and YAC backbone, an origin of transfer, and the selectable marker for _P. tricornutum_ nourseothricin N-acetyl transferase. Interchangeable components include selectable markers for _S. meliloti_: tetracycline, kanamycin/neomycin, and spectinomycin; and origins of replication for _S. meliloti_: the pSymA origin, and pSymB origin. Three vectors were constructed utilizing the repA2B2C2 origin with Spec, Tet or Neo selection, respectively. Three vectors were constructed utilizing the repA1B1C1 origin with Spec, Tet or Neo selection, respectively. **(c)** Following conjugation of three pAGE plasmids from _E. coli_ ECGE101 conjugative strain to _S. meliloti_ RmP4122 ΔpSymA ΔhsdR Nms, an antibiotic selection test for each plasmid was performed. The pAGE1.0, pAGE2.0, and pAGE3.0 vectors in _S. meliloti_ contain antibiotic resistance markers for Spec, Tet, and Neo, respectively. _S. meliloti_ without a plasmid was plated alongside S. meliloti harbouring pAGE1.0, pAGE2.0, or pAGE3.0 on LBmc agar, and LBmc agar supplemented with either spectinomycin, tetracycline, or neomycin.

<br/><img src='https://ryancochrane.github.io/images/paper2figure2.png'>
**Figure 2. Optimized conjugation protocols from _S. meliloti_ to various recipient organisms.** The _S. meliloti_ RmP4098 ΔpSymA ΔhsdR strain used is resistant to streptomycin and requires Fe supplementation for growth. The pAGE1.0 plasmid used confers resistance to spectinomycin. Media supplementation may vary with chosen _S. meliloti_ host and MHS vector. **(a)** Schematic of protocol for conjugation of pAGE1.0 from _S. meliloti_ to _E. coli_. **(b)** Schematic of protocol for conjugation of pAGE1.0 vector from _S. meliloti_ to _S. cerevisiae_. **(c)** Schematic of protocol for conjugation of pAGE1.0 vector from _S. meliloti_ to _P. tricornutum_ –standard protocol. **(d)** Schematic of protocol for conjugation of pAGE1.0 vector from _S. meliloti_ to _P. tricornutum_ – 96-well plate protocol. **(e)** Examples of plates containing transconjugants resulting from conjugation from _S. meliloti_ to _E. coli_, _S. cerevisiae_, and _P. tricornutum_. _P. tricornutum_ transconjugants from the 96-well plate protocol are shown with 100 uL plated on a full plate, and 10 uL spot plated and serially diluted.

<br/><img src='https://ryancochrane.github.io/images/paper2figure3.png'>
**Figure 3. Conjugation efficiency of direct transfer of pAGE1.0 from _S. meliloti_ donor to _E. coli_, _S. cerevisiae_, and _P. tricornutum_ recipients.** Conjugation efficiency for each donor-recipient pair. Post-conjugation non-selective plates and pAGE1.0 selective plates were used to determine the conjugation efficiency, calculated as transconjugants/ recipient, between each donor and recipient pair. Efficiencies are represented as bar plots with three biological and three technical replicates, with error bars illustrating standard error of the mean.

Recommended citation: SL. Brumwell, MR. MacLeod, T. Huang, RR. Cochrane, RS. Meaney, M. Zamani, O. Matysiakiewicz, KN. Dan, P. Janakirama, DR. Edgell, TC. Charles, TM. Finan, BJ. Karas. (2019). "Designer Sinorhizobium meliloti strains and multi-functional vectors enable direct inter-kingdom DNA transfer." <i>PLoS ONE</i>. 14(7): e0219562.
